language/de/cli.php

```diff
@@ -67,6 +68,7 @@
 	'CLI_DESCRIPTION_RECALCULATE_EMAIL_HASH'	=> 'Berechnet die Einträge im Feld user_email_hash der Benutzer-Tabelle neu.',
 	'CLI_DESCRIPTION_SET_ATOMIC_CONFIG'			=> 'Legt den Wert für eine Konfigurations-Einstellung fest, wenn die derzeitige Einstellung dem angegebenen Vergleichswert entspricht',
 	'CLI_DESCRIPTION_SET_CONFIG'				=> 'Legt den Wert für eine Konfigurations-Einstellung fest',
+	'CLI_DESCRIPTION_UPDATE_HASH_BCRYPT'		=> 'Hasht veraltete Passwort-Hashes mit bcrypt und aktualisiert diese beim nächsten Login.',
```

Comment:
Die CLI Funktion macht eigentlich nur ersteres, sie hashed die alten Hashes erneut, so dass alles in bcrypt vorliegt.

Der Login sorgt für den bcrypt hash ohne Zwischenschritt, das macht aber die CLI hier nicht. Daher ist der zweite Satzteil mMn hier unangebracht.